medicines/doc-index-updater/src/delete_manager/mod.rs

```diff
@@ -200,8 +200,10 @@ mod test {
         ))
         .unwrap();
 
+        let result =
+            block_on(state_manager.get_status(removeable_message.get_message().job_id)).unwrap();
```

Comment:
does this need the block_on ?

Reply:
ah because you needed the job id I'd guess

Reply:
`get_status` is async on the trait - test failed without this